EarlGrey/Matcher/GREYMatchers.h

```diff
@@ -488,4 +508,7 @@ GREY_EXPORT id<GREYMatcher> grey_lessThan(id value);
 /** Shorthand for GREYMatchers::matcherForGreaterThan:value. */
 GREY_EXPORT id<GREYMatcher> grey_greaterThan(id value);
 
+/** Shorthand for GREYMatchers::elementAtIndex:index. */
+GREY_EXPORT id<GREYMatcher> grey_elementAtIndex(NSUInteger index);
```

Comment:
GREYMatchers::matcherForElementAtIndex:.

no need to specify the parameter name like this: matcherForGreaterThan:value 

Can you please fix that to matcherForGreaterThan: ? there are a few others in this file that should be fixed.